docs/source/config.rst

```diff
@@ -382,12 +383,79 @@ Postprocessing
    ``error``
       User-defined command to be called if model does not run correctly and
       returns an error code. Useful for automatic error postmortem.
+   
+   ``sync``
+      User-defined command to be called at the start of the ``sync`` pbs job.
+      This is useful for any post-processing before syncing files to a remote
+      archive.
 
 ``postscript``
    This is an older, less user-friendly, method to submit a script after ``payu
    collate`` has completed. Unlike the ``userscripts``, it does not support
    user commands. These scripts are always re-submitted via ``qsub``.
 
+``sync`` 
+   Sync archive to a remote directory using rsync. Make sure that the 
+   configured path to sync output to, i.e. ``path`` is the correct location 
+   before enabling automatic syncing or before running ``payu sync``.
+
+   If postscript is also configured, the latest output and restart files will
+   not be automatically synced after a run.
+
+   ``enable`` (*Default:* ``False``):
+      Controls whether or not a sync job is submitted either after the archive or 
+      collation job, if collation is enabled.
+
+   ``queue`` (*Default:* ``copyq``)
+      PBS queue used to submit the sync job.
+
+   ``walltime``
+      Time required to run the job.
+
+   ``mem`` (*Default:* ``2GB``)
+      Memory required for the job. 
+
+   ``ncpus`` (*Default:* ``1``)
+      Number of ncpus required for the job.
+
+   ``path``
+      Destination path to copy archive outputs to. This must be a unique 
+      absolute path for your experiment, otherwise, outputs will be 
+      overwritten.
+
+   ``restarts`` (*Default:* ``False``)
+      Sync permanently archived restarts, which are determined by 
+      ``restart_freq``.
+
+   ``rsync_flags``
+      Additional flags to add to rsync commands used for syncing files. Note 
+      that these will be added to the default flags ``-vrltoD --safe-links``.
+    
+   ``exclude``
+      Patterns to exclude from rsync commands. This can be a single pattern or 
+      a list of patterns. This is equivalent to rsync's ``--exclude PATTERN``.
```

Comment:
Yes, potentially otherwise it'll may not be valid yaml with special regex characters. I'll add an example to the documentation with patterns with quotes.

Reply:
And also add a comment that patterns need to be quoted if they contain special characters